### PR TITLE
Add edone command to mark Event as done

### PR DESCRIPTION
**Pull request by garyljj**
Added edone command
eg.
`edone 1 2 3`

Add isDone attribute to Event
Add status to Ui: tick for done, cross for not done
Update storage to support isDone

Minor edit to min width in MainWindow to fit Event list nicely (to check: delete ur `preferences.json` file to open with default size)
Width is set to 1041 (weird number as using 1040 caused a one pixel white strip to appear)

### CODE REVIEW

File: src/main/java/seedu/partyplanet/model/event/Event.java [hunk 1]
```diff
@@ -47,6 +60,28 @@ public Remark getDetails() {
...
+    public boolean isDone() {
+        return isDone;
+    }
+
+    /**
+     * Returns event status.
+     * Tick represents done, cross represents not done.
+     */
+    public String getStatus() {
+        return isDone ? "\u2713" : "\u2717";
```
Comment (nickyfoo) [inline src/main/java/seedu/partyplanet/model/event/Event.java:75]: Reminder to change this to not show an X as it's kind of unsightly

**Review by nickyfoo — CHANGES_REQUESTED** [review]
Besides changing the X to an empty string, LGTM

File: src/main/java/seedu/partyplanet/model/event/Event.java (hunk 1 above)
Comment (Ellevy) [inline src/main/java/seedu/partyplanet/model/event/Event.java:75]:
```suggestion
        return isDone ? "\u2713" : "";
```

**Review by Ellevy — APPROVED** [review]
LGTM

**Pull request merged**